classes/shortcodes.php

```diff
@@ -116,32 +119,58 @@ function do_shortcode( $args ) {
 
 				switch ( $address_format ) {
 					case 'town province postalcode' :
-						$full_address = $postmeta['location_address'][0] . ' ' . $postmeta['location_address2'][0] . ' ' . $postmeta['location_city'][0] . $postmeta['location_state'][0] . ' ' . $postmeta['location_zip'][0];
+						$full_address = $postmeta['location_address'][0] . ' '
+						                . $postmeta['location_address2'][0]
+						                . ' ' . $postmeta['location_city'][0]
+						                . $postmeta['location_state'][0] . ' '
+						                . $postmeta['location_zip'][0];
 						break;
 
 					case 'town-province postalcode' :
-						$full_address = $postmeta['location_address'][0] . ' ' . $postmeta['location_address2'][0] . ' ' . $postmeta['location_city'][0] . '-' . $postmeta['location_state'][0] . ' ' . $postmeta['location_zip'][0];
+						$full_address = $postmeta['location_address'][0] . ' '
+						                . $postmeta['location_address2'][0]
+						                . ' ' . $postmeta['location_city'][0]
+						                . '-' . $postmeta['location_state'][0]
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->